SwiftUI_Homework/SwiftUI_Homework/ShopView.swift

```diff
@@ -0,0 +1,173 @@
+//
+//  ShopView.swift
+//  SwiftUI_Homework
+//
+//  Created by Анастасия Козлова on 16.01.2023.
+//
+
+import SwiftUI
+
+/// Экран "Магазин кроссовок"
+struct ShopView: View {
+    
+    // MARK: - Private Constants
+    
+    private enum Constants {
+        static let zeroValue = 0
+        static let trailingValue: CGFloat = 20
+        static let leadingValue: CGFloat = 20
+        static let topValue: CGFloat = 30
+        static let heightValueButton: CGFloat = 50
+        static let widthValueButton: CGFloat = 150
+        static let widthValueImage: CGFloat = 300
+        static let heightValueImage: CGFloat = 200
+        static let sneakersText = "Кроссовки - "
+        static let emptyText = ""
+        static let priceText = "Цена: "
+        static let arrowshapeImageName = "arrowshape.turn.up.forward"
+        static let shareText = "Поделиться"
+        static let basketImageName = "basket"
+        static let addBasketText = "Добавить в корзину"
+        static let questionBasketText = "Вы уверенны, что хотите добавить товар в корзину?"
+        static let addText  = "Добавить"
+        static let notAddText  = "Не добавлять"
+        static let dollarsignImageName = "dollarsign.circle"
+        static let buyText = "Купить"
+        static let questionBuyText =  "Вы уверенны, что хотите совершить покупку?"
+        static let yesText = "Да"
+        static let cancelText = "Отмена"
+        static let infoProductText = "Информация о товаре: "
+    }
+    
+    // MARK: - Public Properties
+    
+    var body: some View {
+        VStack {
+            sneakerModelText
+            Spacer()
+            modelDescriptText
+            sneakerImage
+            sneakerPicker
+            HStack {
+                priceText
+                Spacer()
+                shareButton
+            }
+            HStack {
+                addButton
+                Spacer()
+                buyButton
+            }
+            .padding()
+        }
+    }
+    
+    // MARK: - Private Properties
+    
+    @State private var segmentIndex = 0
+    @State private var isAddShown = false
+    @State private var isBuyShown = false
+    @State private var isShareShown = false
```

Comment:
```suggestion
    @State private var isShareShown = false
    
```
Лучше отделять отступом враперы от обычных свойств